avocado/core/html.py

```diff
@@ -298,7 +298,7 @@ def _render_report(self):
                         template)
             ui.critical("-" * 80)
             ui.critical("%s:\n\n", details)
-            ui.critical("%r\n\n", self.json)
+            ui.critical("%r\n\n", self.result_dict)
```

Comment:
Is this patch really necessary as you're removing the `result_dict` in the next commit?